wagtail/admin/locale/es/LC_MESSAGES/django.po

```diff
@@ -802,19 +802,19 @@ msgstr "Resumen del sitio"
 
 #, python-format
 msgid ""
-"<span>%(total)s</span> Page <span class=\"visuallyhidden\">created in "
```

Comment:
Not sure why this and a few other files weren't replaced when bc6ef435225c5f3a37a5e9ccdcaa59a2cfcb6e2e was done